deblur/deblurring.py

```diff
@@ -36,7 +36,10 @@ def get_sequences(input_seqs):
     seqs = [Sequence(label, seq) for label, seq in input_seqs]
 
     if len(seqs) == 0:
-        raise ValueError("No sequences found!")
+        logger = logging.getLogger(__name__)
+        logger.warn('No sequences found in fasta file!')
```

Comment:
Not trivial remembering the filename we're processing...
There is a ThreadID in the log messages.
If you know how to set a thread name (when it's created) we can set it to the file name and put also the thread name in the log messages